main.tf

```diff
@@ -61,9 +62,11 @@ resource "google_service_account" "org_terraform" {
  ***********************************************/
 
 resource "google_storage_bucket" "org_terraform_state" {
-  project  = module.seed_project.project_id
-  name     = format("%s-%s-%s", var.project_prefix, "tfstate", random_id.suffix.hex)
-  location = var.default_region
+  project            = module.seed_project.project_id
+  name               = format("%s-%s-%s", var.project_prefix, "tfstate", random_id.suffix.hex)
+  location           = var.default_region
+  labels             = var.storage_bucket_labels
+  bucket_policy_only = true
```

Comment:
Should we also consider adding versioning for the state bucket? This recently came up with a team I was working with.

Reply:
Yes, that's a good idea!

Reply:
Resolved by 0c63303562112421c9e52d5be0687dd57c412986.